packages/xo-web/src/xo-app/proxies/deploy-proxy.js

```diff
@@ -268,7 +268,8 @@ const deployProxy = async ({ proxy } = {}) => {
       title,
     }).catch(noop)
     if (choice !== value) {
-      return
+      // throw undefined to interrupt the deployment process and let the ActionButton properly ignore this error
+      throw undefined
```

Comment:
Does it mean they closed the modal? If so, you could simply let `chooseAction` throw.